pkg/restore/restic_restore_action.go

```diff
@@ -107,6 +108,12 @@ func (a *ResticRestoreAction) Execute(input *velero.RestoreItemActionExecuteInpu
 				},
 			},
 		},
+		Resources: corev1.ResourceRequirements{
+			Limits: corev1.ResourceList{
+				corev1.ResourceCPU:    resource.MustParse("100m"),
```

Comment:
We should consider making these optionally configurable via the existing ConfigMap for this plugin.

Reply:
I was thinking about that, but seeing how valid quantities in the wrong fields can break with #1678 gives me pause on the failure mode.

For example, if we allow the user to specify the limits and/or requests and they put `cpu: 128Mi`, it's a valid quantity string, but not for CPU. `MustParse` and `ParseQuantity` only seem to care about the string. So far I don't know that there's a way to find out about that until the pod fails to get scheduled, which I think would cause the Velero Restore to get stuck InProgress.

I'll do some more digging in the Kubernetes code to see if there's a helper function that we may have missed.